justfile

```diff
@@ -9,15 +9,15 @@ _install rootdir="" prefix="/usr/local": _build
 _build-sway:
   {{ just }} sway/cosmic-ext-sway-daemon/build-release
 
-install-sway rootdir="" prefix="/usr/local": build-sway _install
+install-sway rootdir="" prefix="/usr/local": _build-sway _install
   {{ just }} rootdir={{rootdir}} prefix={{prefix}} sway/cosmic-ext-sway-daemon/install
   install -Dm0644 sway/config-cosmic {{rootdir}}/etc/sway/config-cosmic
   install -Dm0644 sway/cosmic-ext-sway.desktop {{rootdir}}{{prefix}}/share/wayland-sessions/cosmic-ext-sway.desktop
   install -Dm0644 sway/start-cosmic-ext-sway {{rootdir}}{{prefix}}/bin/start-cosmic-ext-sway
 
 install-niri rootdir="" prefix="/usr/local": _install
-  install -Dm0644 niri/cosmic-ext-niri.desktop {{rootdir}}{{prefix}}/share/wayland-sessions/cosmic-ext-niri.desktop
-  install -Dm0644 niri/start-cosmic-ext-niri {{rootdir}}{{prefix}}/bin/start-cosmic-ext-niri
+  sudo install -Dm0644 niri/cosmic-ext-niri.desktop {{rootdir}}{{prefix}}/share/wayland-sessions/cosmic-ext-niri.desktop
+  sudo install -Dm0755 niri/start-cosmic-ext-niri {{rootdir}}{{prefix}}/bin/start-cosmic-ext-niri
```

Comment:
Can you please drop the `sudo` additions? We probably have to drop `_build-sway` from the `install-sway` dependencies (and thus maybe remove the underscore), so users can correctly run `just build` and `sudo just install` for system-wide installation. (Which I don't recommend outside of packaging attempts anyway...)